### PR TITLE
added jax.numpy.repeat & jax.numpy.ravel functions

### PR DESCRIPTION
**Pull request by x0axz**
Close https://github.com/unifyai/ivy/issues/12323
Close https://github.com/unifyai/ivy/issues/12451

### CODE REVIEW

File: ivy/functional/frontends/jax/numpy/manipulations.py [hunk 1]
```diff
@@ -36,6 +36,11 @@ def concatenate(arrays, axis=0, dtype=None):
     return ret
 
 
+@to_ivy_arrays_and_back
+def repeat(a, repeats, axis=None, *, total_repeat_length=None):
+    return ivy.repeat(a, repeats, axis=axis)
```
Comment (AnnaTz) [inline ivy/functional/frontends/jax/numpy/manipulations.py:41]: Handling of `total_repeat_length` is also needed, both here and in the test.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/manipulations.py (hunk 1 above)
Comment (x0axz) [reply ivy/functional/frontends/jax/numpy/manipulations.py:41]: @AnnaTz Thanks for the comment. I tried to add this before, but the test for it was failing, and when I removed it, the test passed.

`pytest ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py::test_jax_numpy_repeat`

**Error:**
`repeat() got an unexpected keyword argument 'total_repeat_length'`

Comment (AnnaTz) [reply ivy/functional/frontends/jax/numpy/manipulations.py:41]: Did you try to pass it directly to `ivy.repeat`? I don't think we support this in our api currently. You would have to write a little extra code in you `repeat` function to handle it.

Comment (x0axz) [reply ivy/functional/frontends/jax/numpy/manipulations.py]: please review the changes

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py [hunk 2]
```diff
@@ -176,10 +176,13 @@ def test_jax_numpy_repeat(
 ):
     value_dtype, value = dtype_value
 
+    total_repeat_length = None
+
     if not isinstance(repeat, int):
         repeat_dtype, repeat_list = repeat
         repeat = repeat_list[0]
         value_dtype += repeat_dtype
+        total_repeat_length += repeat
```
Comment (AnnaTz) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_manipulation.py:185]: `total_repeat_length` should be generated inside `@handle_frontend_test` similarly to `repeat`.

**Review by AnnaTz — CHANGES_REQUESTED** [review]

**Comment by x0axz** [comment]
@AnnaTz When I pushed the commit, all these other commits were automatically pushed into my branch; now what should I do?

**Comment by x0axz** [comment]
@AnnaTz I tried to resolve the conflict directly from the branch, and the PR closed automatically. I created a new PR: https://github.com/unifyai/ivy/pull/12795

**Comment by AnnaTz** [comment]
@x0axz this is because of a force push in our codebase. Please refer to this [link](https://discord.com/channels/799879767196958751/983684937712558080/1080080255387189318) for instructions.

**Pull request closed without merging**